### PR TITLE
[Naitei]fix_search_gem_api_rspec

### PR DESCRIPTION
**Pull request by ThaiKhacDuoc**
## Related Tickets
- ticket redmine

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

### CODE REVIEW

**Comment by ThaiKhacDuoc** [comment]
ready

**Comment by ThaiKhacDuoc** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable,
-         :confirmable, :omniauthable, omniauth_providers: [:google_oauth2]
+         :omniauthable, omniauth_providers: [:google_oauth2]
```
Comment (TranMinhHuy2003) [inline app/models/user.rb]: thêm :confirmable

File: app/controllers/api/v1/admin/products_controller.rb [hunk 2]
```diff
@@ -0,0 +1,108 @@
+class Api::V1::Admin::ProductsController < ApplicationController
+  include Pagy::Backend
+
+  before_action :find_product, only: [:show, :update, :destroy]
```
Comment (TranMinhHuy2003) [inline app/controllers/api/v1/admin/products_controller.rb]:
```suggestion
  before_action :find_product, only: %i(show update destroy)
```

**Review by TranMinhHuy2003 — COMMENTED** [review]
cmt

**Comment by ThaiKhacDuoc** [comment]
ready

**Comment by ThaiKhacDuoc** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**